app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -95,11 +94,17 @@ def throttled_url
 
       def throttled_else_increment
         Throttle.for(
-          target: user_id,
+          user: effective_user,
           throttle_type: :idv_acuant,
         ).throttled_else_increment?
       end
 
+      # Ideally we would not have to re-implement the EffectiveUser mixin
+      # but flow_session sometimes != controller#session
+      def effective_user
+        current_user || User.find(user_id)
```

Comment:
Since `user_id` is checking again for `current_user`, could consider skipping straight to `user_id_from_token`

```suggestion
        current_user || User.find(user_id_from_token)
```